tests/TestCase.php

```diff
@@ -9,27 +9,51 @@
  * file that was distributed with this source code.
  */
 
-class TestCase extends \PHPUnit\Framework\TestCase 
+class TestCase extends \PHPUnit\Framework\TestCase
 {
     protected $client;
     const HOST = "http://localhost:1337";
 
-    public function setUp() {
+    public function setUp()
+    {
         `php ../run.php`;
         sleep(0.1);
         $this->client = new \GuzzleHttp\Client();
+        
+        $body = $this->get('/founder');
+        if (!isset($body["id"])) {
+            $this->markTestSkipped('Can not get founder id');
+            return;
+        };
+        $this->founder_id = $body["id"];
     }
 
-    public function tearDown() {
+    public function tearDown()
+    {
         //unset($this->graph);
     }
-    
-    protected function get(string $path, bool $headers=false) {
-        $res = $this->client->request('GET', self::HOST.$path);
-        if($headers)
+
+    protected function get(string $path, bool $headers = false)
+    {
+        $res = $this->client->request('GET', self::HOST . $path);
+        if ($headers) {
             return $res;
+        }
+
         $body = json_decode($res->getBody(), true);
         return $body;
     }
 
-}
\ No newline at end of file
+    protected function post(string $path, array $postData)
```

Comment:
i would add $headers similarly to the get function and make them behave similarly, for consistency purposes.